conf/gulp-tasks/library.gulpfile.js

```diff
@@ -46,8 +46,8 @@ function createJSBundlesForLanguages (languages) {
   });
 }
 
-exports.default = function defaultJSBundles () {
-  return createJSBundlesForLanguages(DEFAULT_LOCALES);
+exports.default = function defaultLanguageJSBundle () {
```

Comment:
Minor nit, but I think we want the suffix to be `JSBundles` (as more than one is created).

Reply:
Ah, I was thinking singular because this now single-language. Updating